### PR TITLE
closes  #7. Migrating the Bugzilla Workshop from https://github.com/OSLC/integrating-products-with-oslc-tutorial

### LINKED ISSUE #7: Integrate the Bugzilla Workshop into this "OSLC Developer Guide" site
1.
The md source for the Bugzilla worshop (https://archive.open-services.net/resources/tutorials/integrating-products-with-oslc/) is under https://github.com/OSLC/integrating-products-with-oslc-tutorial.

Is it better to migrate this content into this repo? 
This also implies we no longer need to refer to https://archive.open-services.net/resources/tutorials/integrating-products-with-oslc/.

2. At the very least, we update the content to refer to the proper location of the source code.
https://github.com/eclipse/lyo.docs/tree/master/lyo-rest-workshop

### PR DESCRIPTION
**Pull request by jadelkhoury**
closes  #7. Migrating the Bugzilla Workshop from https://github.com/OSLC/integrating-products-with-oslc-tutorial

### CODE REVIEW

**Comment by jadelkhoury** [comment]
Status: I have done a thourough check of the links. Seems quite stable now, except for the  /integrating_products_with_oslc/running_the_examples page.

What is left to do. First point is most critical:

- [x] Locate the source code git://git.eclipse.org/gitroot/lyo/org.eclipse.lyo.docs.git. https://github.com/eclipse/lyo.docs does not seem to match, if one sees for example the package names used.
- [x] Finalize and simplify the page /integrating_products_with_oslc/running_the_examples
- [x] Update the references/links to the tutorial from all over this DeveloperGuide to point to this new content.
- [x] Update Readme file at https://github.com/OSLC/integrating-products-with-oslc-tutorial to point to Developer Guide.
- [x] Update links/references at open-services.net to point Developer Guide.
- [x] Update Readme.md file at https://github.com/OSLC/integrating-products-with-oslc-tutorial to point to DeveloperGuide.

**Comment by berezovskyi** [comment]
Are you blocked by any other issue? I suggest removing the label if not. It is to indicate that something else is blocking you.

**Comment by jadelkhoury** [comment]
I’m blocked because I can’t find the source code. Where is the old git://git.eclipse.org/gitroot/lyo/org.eclipse.lyo.docs.git?

**Comment by berezovskyi** [comment]
I see, sorry! https://github.com/eclipse/lyo.docs



--
/Andrew
(from phone)



On Wed, May 22, 2019 at 7:48 AM +0200, "Jad El-khoury" <notifications@github.com<mailto:notifications@github.com>> wrote:


I’m blocked because I can’t find the source code. Where is the old git://git.eclipse.org/gitroot/lyo/org.eclipse.lyo.docs.git?

**Comment by jadelkhoury** [comment]
> I see, sorry! https://github.com/eclipse/lyo.docs

That's what I first htought, but something tells me it is not hte same code. I will look into Lab6 in details and see if it is good enough.

@berezovskyi what version of Lyo do you think we should attempt to specify in the pom files? 2.4.0 sounds too risky. What is the most stable and earlier versions we have released?

I just want to make this tutorial public for now. I don't care if it follows the latest development - yet.

**Comment by berezovskyi** [comment]
2.3.0 should be safe. It was used by people for almost a year without any major bug reports.

**Comment by jadelkhoury** [comment]
I'm pretty happy with the final result. So, if anyone wants to do a final walkthrough, I am happy to merge.

**Comment by jadelkhoury** [comment]
I have forgotten, do I need to compile the html pages locally, or is that taken care of at the server end?

**Comment by berezovskyi** [comment]
https://github.com/OSLC/oslc-site-hugo/blob/master/content/resources/eclipse-lyo.md

**Comment by berezovskyi** [comment]
what do you think? the URL will remain the same for search engine reasons.

**Comment by berezovskyi** [comment]
you need to run the deploy script in the root folder

**Comment by berezovskyi** [comment]
just deployed

**Comment by jadelkhoury** [comment]
> 
> 
> what do you think? the URL will remain the same for search engine reasons.

@berezovskyi! You are on the wrong issue.

**Comment by berezovskyi** [comment]
oh yes I am

**Comment by berezovskyi** [comment]
> I have forgotten, do I need to compile the html pages locally, or is that taken care of at the server end?

GH will take care of everything

File: _harp/_data.json [hunk 1]
```diff
@@ -14,8 +14,8 @@
   "./eclipse_lyo/eclipse-lyo" : {
     "title": "Tools & SDKs for Java Developers - Eclipse Lyo"
   },
-  "sample-applications-and-code": {
-    "title": "Sample applications and code"
+  "tutorials-and-sample-applications": {
```
Comment (berezovskyi) [inline _harp/_data.json]: aghh, this will break links and kick us off the (already low) positions in Google Search...

**Review by berezovskyi — APPROVED** [review]

**Comment by berezovskyi** [comment]
@jadelkhoury I forgo, you actually have to re-compile Harp site first and then Github will publish the HTML automatically.

**Comment by jadelkhoury** [comment]
@berezovskyi 
I fixed the borken link to "sample-applications-and-code". Plus, regenerated the html. 
Many files were generated it seems, but I think it is safe.

Anything odd you can see? Otherwise, ready to merge.

**Comment by berezovskyi** [comment]
Look good, please go ahead @jadelkhoury

**Pull request merged**